### ISSUE TITLE
Suggestion - open source this project under some free license

### ISSUE THREAD

**Issue by C0rn3j** [nanopool/ewbf-miner#20]
So far I tested silentarmy, ccminer and nheqminer, they are all slower than ewbf-miner.

I'd love this project to become open-source as it seems to be one of the best miners.

The dev fee that is by default set to 2% can even be turned off apparently, which is awesome.

Since I can turn it off, I don't see why this project would be beneficial to leave as closed-source - you could only possibly gain from community improvements. It'd also spread awareness about nanopool, which, again, would get you money from people using your pool.

So please consider releasing this under some copyleft license like the GPL, thanks!

**Comment by Moordrik** [comment]
I agree that by far this is the best miner for Nvidia GPU's based on my testing as well.  I use it for my windows based miners and my Linux based miner and get great results.  Even with the 2% dev feed it runs circles around nheqminer in my experiences (which don't get me wrong, it's top notch as well). Because of this I'm good with it staying closed simply because it protects what has been done from being duplicated by someone else and the work losing all value.  Why would someone pay a 2% dev fee if miner X just took what made it special and made a miner without a dev fee.  I think that is why it's not GPL, it gives them the revenue stream to support the updates and refinements that keep this miner moving forward.  They do give you the option to opt out of the dev fee, but my understanding is it will operate a slightly reduced rate (understandable).  I leave it on.  I don't even notice the dev shares in my sols/s or hash rate (whichever you go by).  I just want this project to keep going, and improving.  It's a win/win for miner and dev.

Background:  I'm just a miner, I'm not big time, I've got 2 gaming rigs with 1070 GTX's, and I'm building a 8 GPU Linux rig.  All of them are using EWBF now and I have ZERO complaints with it.

**Comment by Lightn1ng** [comment]
@Moordrik 

> I'm not big time

> I'm building a 8 GPU Linux rig

pick one

**Comment by Moordrik** [comment]
Are you saying I'm a "big" miner?  If you think a single 8 GPU rig is big time your sense of scale is skewed.  One 8 GPU rig only generates ~ 2900 Sols/s.  Compared to the "big" miners I see out there who average from 60k-300k sols/s I'm "small".  I'm not even a drop in the bucket compared to their output.  If I had maybe 4-6 rigs, then I would call myself at least a "serious" miner.  What I'm attempting I would call dabbling at this point, small amount of risk.

**Comment by Lightn1ng** [comment]
@Moordrik I just find it amusing to say you're not big time then say you're building something that would seem very big-time to the majority of miners who are mining on their gaming GPU in their free time every so often.

**Comment by Moordrik** [comment]
I guess I can see your point.  I started that way, mining on the 2 1070's one in both gaming computers (mine and the wife).  That was my "test" to get my feet wet and see what I could do.  Then came what I am doing atm.  This is a proof of concept with minimal risk for me.  If it fails I'm at most out ~3300 USD.  But I can always repurpose the gear and/or sell the cards to recoup some if not all my money (right now the 1070's are selling for +200 USD what I paid).  Worst case, it's a learning experience.  It's new, I'm having fun with it, so it's a win in my book regardless of the outcome.
